library/systemd/src/lib/yast2/systemd_unit.rb

```diff
@@ -191,16 +242,14 @@ def state_name_enabled?(state)
       end
 
       def extract_properties
-        systemd_unit.input_properties.each do |name, property|
+        systemd_unit.propmap.each do |name, property|
           self[name] = raw[/#{property}=(.+)/, 1]
         end
       end
 
       def load_systemd_properties
-        properties = systemd_unit.input_properties.map do |_, property_name|
-          " --property=#{property_name} "
-        end
-        systemd_unit.command("show", options: properties.join)
+        names = systemd_unit.propmap.values
+        systemd_unit.command("show", options: "--property=" + names.join(","))
```

Comment:
just to be sure, how name with whitespace is handled? or it cannot happen?

Reply:
No, property names do not have whitespace.